_config.yml

```diff
@@ -641,3 +641,18 @@ jsonresume:
   - languages
   - interests
   - references
+
+# Customize the section titles in the cv page, leave it blank for default
```

Comment:
Add to this comment the info about it only being used when the cv is loaded from jsonr. Even though it is in that section in _config.yml, it's better to make it clear.